README.md

```diff
@@ -0,0 +1,17 @@
+# Funcational tests scaffolding
+
+[![Build Status](https://travis-ci.org/keboola/functional-tests-scaffolding.svg?branch=master)](https://travis-ci.org/keboola/functional-tests-scaffolding)
+
+# Usage
+
+`vendor/bin/functional`
```

Comment:
Popsat jak má vypadat ta struktura tech adresářů? Koukám že už to např. umí testovat chyby pomocí `expceted` file?

Reply:
ne, to je akorat, ze kdyz tam neni expected, tak to ocekava fail

Reply:
aha, dik. Nevedel jsem ze `file_exists` kontroluje i adresar

Reply:
https://github.com/keboola/functional-tests-scaffolding/blob/tf2-inital-import/README.md